src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/EvalResultDataItem.cs

```diff
@@ -21,13 +33,13 @@ namespace Microsoft.CodeAnalysis.ExpressionEvaluator
     /// </remarks>
     internal sealed class EvalResultDataItem : DkmDataItem
     {
-        /// <summary>
-        /// May be null for synthesized nodes like "Results View", etc.
-        /// </summary>
-        public readonly string NameOpt;
+        public readonly ExpansionKind Kind;
```

Comment:
Parent:  We also need parent to decide if this is a "root level" node.  (we don't respect debugger display name on the root level).

Kind:  Yes, it affects those, but it also decides whether or not we respect debugger display.  I can squint and see how we might change Kind -> {DisplayName, DisplayValue, DisplayTypeName, IgnoreDebuggerDisplayAttribute}...but it might take some work (I'd be inclined to file an issue rather than holding off on the API change unless you have strong feelings about "Kind")...